common/ledger/dataformat/dataformats.go

```diff
@@ -9,16 +9,19 @@ package dataformat
 import "fmt"
 
 const (
-	// Version1x specifies the data format in version 1.x
-	Version1x = ""
+	// PreviousVersion specifies the data format in previous version (1.x for now)
+	PreviousVersion = ""
```

Comment:
```suggestion
	LastMajorVersion = ""
```
This is because both 2.0 and 2.1 are called as two different versions. It would be good if we can be explicit.

Reply:
These versions are not as such related to fabric versions. As you see the package name is `dataformat`, so they would change only when the data format is changed.

Reply:
@cendhu Thank you for your comments. As @manish-sethi  said, these versions are for the data formats in ledger databases (e.g, dataformat.CurrentVersion). They may or may not change when fabric version is upgraded. Currently, we don't have the concept for "major" data format version.

Reply:
I see. Thanks. I think the confusion was due to the comment `CurrentVersion specifies the data format in the current version (2.0 for now)` and we have `CurrentVersion = "2.0"`. In that case, can we rename `CurrentVersion` to `CurrentDataVersion` and `PreviousVersion` to `PreviousDataVersion`.   Even some other name is fine but let's be more explicit to avoid hidden meaning. I think it is better to remove `2.0` as well but we cannot do that now, right? Some of the networks that have migrated to 2.0 would already have this format. Usually, `2.x` can be interpreted as 2 -- major version, x -- minor version. I assume we don't support major.minor version for the data format. When we change the data format in the future, we can use non-decimal number to avoid these confusions.

Reply:
I am fine with the names `CurrentDataVersion` and `PreviousDataVersion`, if @wenjianqiao is fine. 
Yes, version is just a string for us and we cannot change `2.0`. The intention was to keep track when this format was introduced than anything else. So, in a way, that a binary is expected to work with the greatest data format only where  `data format version <= binary version`. @wenjianqiao - may be some of this description, you may like to add in the data format file.

Reply:
I am fine with the names CurrentDataVersion and PreviousDataVersion

Reply:
Talked offline and agreed to change the constants to CurrentFormat and PreviousFormat.